web/pandas_web.py

```diff
@@ -418,6 +418,9 @@ def main(
                 body = markdown.markdown(
                     content, extensions=context["main"]["markdown_extensions"]
                 )
+                # Apply Bootstrap's table formatting manually
+                # Python-Markdown doesn't let us config table attributes by hand
+                body = body.replace("<table>", '<table class="table table-bordered">')
```

Comment:
Really nice. At some point I thought on creating a plugin for python markdown to do this (the core dev doesn't want to add anything into the project, which is probably good). But didn't think about this as a workaround.